supervisor/dbus/core.py

```diff
@@ -0,0 +1,93 @@
+"""D-Bus interface objects."""
+import logging
+from typing import List
+
+from ..const import SOCKET_DBUS
+from ..coresys import CoreSys, CoreSysAttributes
+from .agent import OSAgent
+from .hostname import Hostname
+from .interface import DBusInterface
+from .logind import Logind
+from .network import NetworkManager
+from .rauc import Rauc
+from .systemd import Systemd
+from .timedate import TimeDate
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+class DBusManager(CoreSysAttributes):
```

Comment:
This is also called manager, is core the best name?
ref: https://github.com/home-assistant/supervisor/pull/2940#discussion_r659724749